pkg/cmd/proxy/health/exec.go

```diff
@@ -150,8 +150,10 @@ func (o *Options) run(streams genericclioptions.IOStreams) error {
 
 	w := newWriter(streams)
 	for _, cluster := range managedClusterList.Items {
-		if err := o.visit(&w, hubRestConfig, addonClient, tunnel.DialContext, cluster.Name); err != nil {
-			klog.Errorf("An error occurred when requesting: %v", err)
+		if len(o.clusters) == 0 || contains(o.clusters, cluster.Name) {
```

Comment:
how about putting o.clusters to a sets.String?

Reply:
updated